notebooks/examples/python/Ship2ShipTransfers/00. Data Prep.py

```diff
@@ -0,0 +1,200 @@
+# Databricks notebook source
+# MAGIC %md ## We First Prep the data and download it
+
+# COMMAND ----------
+
+import mosaic as mos
+
+spark.conf.set("spark.databricks.labs.mosaic.geometry.api", "ESRI")
+spark.conf.set("spark.databricks.labs.mosaic.index.system", "H3")
+mos.enable_mosaic(spark, dbutils)
+
+# COMMAND ----------
+
+# MAGIC %md ##AIS Data
+
+# COMMAND ----------
+
+dbutils.fs.mkdirs("/tmp/vessels")
+
+# COMMAND ----------
+
+# MAGIC %sh
+# MAGIC # see: https://coast.noaa.gov/htdata/CMSP/AISDataHandler/2018/index.html
+# MAGIC # we download data to dbfs:// mountpoint (/dbfs)
+# MAGIC cd /dbfs/tmp/vessels/
+# MAGIC
+# MAGIC wget -np -r -nH -L --cut-dirs=3 https://coast.noaa.gov/htdata/CMSP/AISDataHandler/2018/AIS_2018_01_31.zip > /dev/null 2>&1
+# MAGIC unzip 2018/AIS_2018_01_31.zip
+
+# COMMAND ----------
+
+schema = """
+  MMSI int, 
+  BaseDateTime timestamp, 
+  LAT double, 
+  LON double, 
+  SOG double, 
+  COG double, 
+  Heading double, 
+  VesselName string, 
+  IMO string, 
+  CallSign string, 
+  VesselType int, 
+  Status int, 
+  Length int, 
+  Width int, 
+  Draft double, 
+  Cargo int, 
+  TranscieverClass string
+"""
+
+AIS_df = (
+    spark.read.option("badRecordsPath", "/tmp/ais_invalid")  # Quarantine bad records
+    .csv("/tmp/vessels/2018", header=True, schema=schema)
+    .filter("VesselType = 70")  # Only select cargos
+    .filter("Status IS NOT NULL")
+)
+display(AIS_df)
+
+# COMMAND ----------
+
+# MAGIC %sql
+# MAGIC CREATE DATABASE IF NOT EXISTS ship2ship
+
+# COMMAND ----------
+
+(AIS_df.write.format("delta").mode("overwrite").saveAsTable("ship2ship.AIS"))
+
+# COMMAND ----------
+
+# MAGIC %md ## Continental US Data
+
+# COMMAND ----------
+
+from pyspark.sql.functions import *
+
+one_metre = 0.00001 - 0.000001
+tolerance = 500 * one_metre
```

Comment:
can we add some context around why we chose 500m here?

Reply:
I have deleted this section since this approach is not used later.